<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>80</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>id magasin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>70</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom Magasin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>70</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Num Article</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>70</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom Article</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>60</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix Article</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>90</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre Article</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_idmagasin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>14</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nommagasin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>64</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NumArticel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>114</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nomarticle">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>14</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_prixarticle">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>64</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nmbrarticle">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>114</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>640</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_afficher">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Ajouter">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modifier">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_supprimer">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_recherche">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>70</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Recherche :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Trier</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tri par Num Article</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tri par Nom Article</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tri par Prix Article</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_statestique">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Statestique</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PDF">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>chat1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>chat2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_QRcode">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>190</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>QRcode</string>
    </property>
   </widget>
   <widget class="QLabel" name="qr_code">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>301</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_camera">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PDF_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>460</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_whattsap">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Whattsap</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
